tuf-spec.md

```diff
@@ -1586,12 +1586,12 @@ We now explain how a repository should write metadata and targets to
 produce self-contained consistent snapshots.
 
 Simply put, TUF should write every metadata file as such: if the
```

Comment:
I think this and the below paragraph should also be MUSTs.

Reply:
I've added quite a few more RFC 2119 keywords in 6c54aa8 and cleaned up other paragraphs in the section.